pkg/skuba/actions/node/remove/remove.go

```diff
@@ -31,17 +33,27 @@ import (
 )
 
 // Remove removes a node from the cluster
-func Remove(target string, drainTimeout time.Duration) error {
-	client, err := kubernetes.GetAdminClientSet()
-	if err != nil {
-		return errors.Wrap(err, "unable to get admin client set")
-	}
-
+func Remove(client clientset.Interface, target string, drainTimeout time.Duration) error {
 	node, err := client.CoreV1().Nodes().Get(target, metav1.GetOptions{})
 	if err != nil {
 		return errors.Wrapf(err, "[remove-node] could not get node %s", target)
```

Comment:
if the target node does not exist, what's the command line output to the user?

Reply:
```bash
** This is a BETA release and NOT intended for production usage. **
F0722 09:47:23.559485   16812 remove.go:52] error removing node master-100: [remove-node] could not get node master-100: nodes "master-100" not found
```